react/index.js

```diff
@@ -3,6 +3,11 @@ import { ExtensionPoint } from 'render'
 
 // FIXME: Delete when we can use true data.
```

Comment:
Is it time? 😄

Reply:
If I delete it, we may not see the component working properly... So... Delete this?